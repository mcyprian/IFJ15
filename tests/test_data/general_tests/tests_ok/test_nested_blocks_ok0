int main()
{
    string c = "ahoj1";
    cout << c << "\n";

    {
        string c = "ahoj2";
        cout << c << "\n";
        {
            string c = "ahoj3";
            cout << c << "\n";
            {
                string c = "ahoj4";
                cout << c << "\n";
                {
                    string c = "ahoj5";
                    cout << c << "\n";
                }
                cout << c << "\n";
            }
            cout << c << "\n";
        }
        cout << c << "\n";
    }
    
    cout << c << "\n";

    return 0;
}
