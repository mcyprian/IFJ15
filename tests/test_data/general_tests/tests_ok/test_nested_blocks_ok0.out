ahoj1
ahoj2
ahoj3
ahoj4
ahoj5
ahoj4
ahoj3
ahoj2
ahoj1
